common/lib/capa/capa/tests/response_xml_factory.py

```diff
@@ -699,6 +699,8 @@ def create_response_element(self, **kwargs):
 
             *additional_answers*: list of additional asnwers.
 
+            *old_answers*: list of additional answers to be coded in old format
```

Comment:
hmm really don't like the idea of there just being an "old" format. that's so wishy washy, and subject to time depreciation in a few months/years.